src/main/java/com/jcabi/github/RtContents.java

```diff
@@ -39,6 +39,8 @@
 import javax.json.Json;
 import javax.json.JsonObject;
 import javax.json.JsonStructure;
+import javax.json.JsonValue;
+
```

Comment:
@sadovnikov This line should not be empty. Please see [this](https://ci.appveyor.com/project/yegor256/jcabi-github/build/485).